### PR TITLE
Fix missing dependency for portal directive and  mismatch in reference getter/setter types

### PR DESCRIPTION
**Pull request by RasmusKjeldgaard**
## Which issue does this PR close?

This PR closes no issue but fixes two compile-time problems that occur when installing Kirby v8.3.0:
image

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [ ] Determine if your changes are a fix, feature or breaking-change, and add the matching label to your PR. If it is tooling, dependency updates or similar, add ignore-for-release.
- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [ ] Request that the changes are code-reviewed 
- [ ] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: libs/designsystem/shared/floating/src/floating.directive.ts [hunk 1]
```diff
@@ -402,6 +402,7 @@ export class FloatingDirective implements OnInit, OnDestroy {
   }
 
   private handleClickOutsideHostElement(event: Event): void {
+    if (!(event.target instanceof HTMLElement)) return;
```
Comment (jakobe) [inline libs/designsystem/shared/floating/src/floating.directive.ts]: Why this guard? What can be clicked that's not an `HTMLElement`? 🤔

Comment (RasmusKjeldgaard) [reply libs/designsystem/shared/floating/src/floating.directive.ts]: event.target is type `EventTarget`, so I figured this early return was the easiest way to make typechecking behave on the next line:
```
const clickedOnReferenceWithClickTriggerEnabled: boolean =
      this.reference?.nativeElement.contains(event.target)
```

Comment (RasmusKjeldgaard) [reply libs/designsystem/shared/floating/src/floating.directive.ts]: Definitely open to suggestions for a more slick version of this!

Comment (jakobe) [reply libs/designsystem/shared/floating/src/floating.directive.ts]: As `this.reference?.nativeElement.contains` takes a `Node` I don't see a reason for this guard / type check...

Comment (RasmusKjeldgaard) [reply libs/designsystem/shared/floating/src/floating.directive.ts]: Whoops, I edited my original comment now meant to say `EventTarget` not `Node`

**Review by jakobe — APPROVED** [review]

**Pull request merged**